source/contracts/extensions/MarketFeeCalculator.sol

```diff
@@ -68,27 +92,24 @@ contract MarketFeeCalculator {
     }
 
     function getRepMarketCapInAttoeth(IUniverse _universe) constant public returns (uint256) {
-        // TODO: get these from an auto-generated market
+        // TODO: get this from an auto-generated market or some other special contract
         uint256 _attorepPerEth = 11 * 10 ** 18;
         uint256 _repMarketCapInAttoeth = _universe.getReputationToken().totalSupply() * _attorepPerEth;
         return _repMarketCapInAttoeth;
     }
 
     function getTargetRepMarketCapInAttoeth(IReportingWindow _reportingWindow) constant public returns (uint256) {
-        uint256 _outstandingSharesInAttoeth = getOutstandingSharesInAttoeth(_reportingWindow);
-        uint256 _targetRepMarketCapInAttoeth = _outstandingSharesInAttoeth * 5;
-        return _targetRepMarketCapInAttoeth;
+        IUniverse _universe = _reportingWindow.getUniverse();
+        return _universe.getOpenInterestInAttoEth() * TARGET_REP_MARKET_CAP_MULTIPLIER;
     }
 
-    function getOutstandingSharesInAttoeth(IReportingWindow) constant public returns (uint256) {
-        // TODO: start tracking the real number and store it somewhere
-        // NOTE: make sure we are getting the share value in attoeth, since complete set fees are not normalized across markets
-        // NOTE: eventually we will need to support shares in multiple denominations
-        uint256 _outstandingSharesInAttoeth = 100 * 10 ** 18;
-        return _outstandingSharesInAttoeth;
+    function getPreviousReportingWindow(IReportingWindow _reportingWindow) constant private returns (IReportingWindow) {
```

Comment:
I think there is some other place we are doing this as well.  I wonder if we should just put a function on `ReportingWindow` called `getPreviousReportingWindow` and `getNextReportingWindow` so we don't have to keep re-implementing this logic?

Reply:
That seems like a good idea. Will do